src/ProductConstructionService/Readme.md

```diff
@@ -48,8 +48,21 @@ We're using a Managed Identity to authenticate PCS to BAR. You'll need to run th
  - CREATE USER [`ManagedIdentityName`] FROM EXTERNAL PROVIDER
  - ALTER ROLE db_datareader ADD MEMBER [`ManagedIdentityName`]
  - ALTER ROLE db_datawriter ADD MEMBER [`ManagedIdentityName`]
+If the service is being recreated and the same Managed Identity name is reused, you will have to drop the old MI from the BAR, and then run the SQL queries above
```

Comment:
I think this might be instead saying "the same MI is not reused"?

Reply:
No, if we delete an Identity, and then recreate it with the same name, it won't just work. This is because, when we're creating SQL user for the identity, it looks up Entra, checks if it's the only identity with that name, and stores it's ID somewhere. So when we recreate the identity with the same name, the IDs won't match, and the auth will fail